### ISSUE TITLE
Reverse operation of `:lift-ns-in-code?` and `:lift-ns?` config options

### ISSUE THREAD

**Issue by mynomoto** [kkinnear/zprint#80]
I don't really like the new map syntax and would like that the formatter did the opposite of what those options do. Something like `unlift-ns?` to convert maps using the new syntax to the old syntax. Is that something desirable for the project?
The new map syntax also breaks when the code is run on old versions of clojure.

**Comment by kkinnear** [comment]
Well, I don't see why not.  While I'm pretty sure I know what you want, could you give me at least one "before" and "after" example of what you want?  I don't want to go off and implement something and find out I had it wrong.  That's always depressing.

**Comment by mynomoto** [comment]
Using the examples of `lift-ns?` that would be:
```
(zprint #:x{:a :b, :c :d} {:map {:unlift-ns? true}})
{:x/a :b :x/c :d}

(zprint #:zprint.core{:a :b, :c :d} {:map {:unlift-ns? true}})
{:zprint.core/a :b :zprint.core/c :d}
```

**Comment by kkinnear** [comment]
This is implemented in version 0.4.16.  I hope that it does what you want!
